Week3/Week3-JS-I.js

```diff
@@ -27,129 +27,124 @@ const newModulo = 21 % 5 === null;
 
 // 7
 function returnString(str) {
-  //simply return the string provided: str
+  return str;
 }
 
 // 8
 function add(x, y) {
-  // x and y are numbers
-  // add x and y together and return the value
-  // code here
+  return x + y;
 }
 
 // 9
 function subtract(x, y) {
-  // subtract y from x and return the value
-  // code here
+  return x - y;
 }
 
 // 10
 function multiply(x, y) {
-  // multiply x by y and return the value
-  // code here
+  return x * y;
 }
 
 // 11
 function divide(x, y) {
-  // divide x by y and return the value
-  // code here
+  return x % y;
 }
 
 // 12
 function areEqual(x, y) {
-  // return true if x and y are the same
-  // otherwise return false
-  // code here
+  if (x === y) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 13
 function areSameLength(str1, str2) {
-  // return true if the two strings have the same length
-  // otherwise return false
-  // code here
+  if (str1.length === str2.length) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 14
 function lessThanNinety(num) {
-  // return true if the function argument: num , is less than ninety
-  // otherwise return false
-  // code here
+  if (num < 90) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 15
 function greaterThanFifty(num) {
-  // return true if num is greater than fifty
-  // otherwise return false
-  // code here
+  if (num > 50) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 16
 function getRemainder(x, y) {
-  // return the remainder from dividing x by y
-  // code here
+  return x % y;
```

Comment:
What is this operator? And what does it do?

Reply:
The remainder operator and it divides the two numbers and gives you the remainder.

Reply:
Awesome. It is called the modulo operator (in dev terms) excellent!